src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.DefaultOutputPaths.targets

```diff
@@ -11,29 +11,109 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-   <!--
-    Apply the same default output paths as Microsoft.Common.targets now since we're running before them,
-    but need to adjust them and/or make decisions in terms of them.
-
-    Also note that common targets only set a default OutputPath if neither configuration nor
+  <!--
+    Note that common targets only set a default OutputPath if neither configuration nor
     platform were set by the user. This was used to validate that a valid configuration is passed,
     assuming the convention maintained by VS that every Configuration|Platform combination had
     an explicit OutputPath. Since we now want to support leaner project files with less
-    duplication and more automatic defaults, we always set a default OutputPath and can no
-    longer depend on that convention for validation. Getting validation re-enabled with a
-    different mechanism is tracked by https://github.com/dotnet/sdk/issues/350
+    duplication and more automatic defaults, we always set a default OutputPath.
    -->
-  <PropertyGroup>
-    <Configuration Condition="'$(Configuration)'==''">Debug</Configuration>
-    <Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
-    <PlatformName Condition="'$(PlatformName)' == ''">$(Platform)</PlatformName>
 
+  <!-- Projects which don't use Microsoft.NET.Sdk will typically define the OutputPath directly (usually in a
+       Configuration-specific PropertyGroup), so in that case we won't append to it by default. -->
+  <PropertyGroup Condition="'$(UsingNETSdkDefaults)' == 'true'">
```

Comment:
I believe this is a breaking change to existing paths? Do we want this for the existing behavior as well?

Reply:
This is behavior which already existed, it's just been moved around to a different .targets file.  I believe there shouldn't be any breaking change with this.

Reply:
You're right, there is no breaking change. The first concern would be rid inference but this is moving the change to earlier in evaluation (defaultoutpathtargets) instead of later (after ridinference.targets). Thanks!